package.json

```diff
@@ -57,7 +57,8 @@
     "underscore": "~1.8.2",
     "vinyl-source-stream": "~1.1.0",
     "watchify": "~2.4.0",
-    "when": "~3.7.2"
+    "when": "~3.7.2",
+    "d3":"~3.5.5"
```

Comment:
can you reorder this (and add a space)? `npm install --save-dev d3` should do the trick. Otherwise, whenever someone installs a package using `npm install --save*` the whole file will get reformatted again.